Clarify instructions (#36281)
dotnet/docs: docs/core/tutorials/debugging-with-visual-studio-code.md

The breakpoint is located after a `Console.ReadLine` method call. The **Debug Console** doesn't accept terminal input for a running program. To handle terminal input while debugging, you can use the integrated terminal (one of the Visual Studio Code windows) or an external terminal. For this tutorial, you use the integrated terminal.

> 1. The project folder contains a .vscode folder. Open the *launch.json* file that is located in the *.vscode* folder.

Author: ```suggestion
1. The project folder contains a *.vscode* folder. Open the *launch.json* file that is in the *.vscode* folder.
```

> 1. Change the `console` setting from `internalConsole` to `integratedTerminal`:

Author: ```suggestion
1. In *launch.settings*, change the `console` setting from `internalConsole` to `integratedTerminal`:
```

The breakpoint is located after a `Console.ReadLine` method call. The **Debug Console** doesn't accept terminal input for a running program. To handle terminal input while debugging, you can use the integrated terminal (one of the Visual Studio Code windows) or an external terminal. For this tutorial, you use the integrated terminal.

> 1. The project folder contains a *.vscode* folder. Open the *launch.json* file that is in the *.vscode* folder.

Reviewer: ```suggestion
1. The project folder contains a *.vscode* folder. Open the *launch.json* file that's in the *.vscode* folder.
```

> 1. In *launch.settings*, change the `console` setting from `internalConsole` to `integratedTerminal`:

Reviewer: ```suggestion
1. In *launch.json*, change the `console` setting from `internalConsole` to `integratedTerminal`:
```
